_haikus/unnamed.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: haiku
+title: unnamed
+author: Stoozy
+---
+the selected file <br>
```

Comment:
This line has one too many syllables, perhaps try:
```
the chosen file <br>
```

Reply:
The original was "The chosen file". When I ran npm test on it, it said there were 4 syllables instead of 5 and failed. So I put selected instead.